{
  "id": "EUVD-2019-10698",
  "enisaUuid": "990f6366-02cb-394a-90d7-9d45b8cdebca",
  "description": "An issue was discovered in GitLab Community Edition (CE) and Enterprise Edition (EE) 12.3 through 12.6.1. It allows Denial of Service.",
  "datePublished": "Jan 13, 2020, 8:04:21 PM",
  "dateUpdated": "Aug 5, 2024, 2:39:07 AM",
  "baseScore": 4.3,
  "baseScoreVersion": "3.1",
  "baseScoreVector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:L",
  "references": [
    "https://about.gitlab.com/blog/categories/releases/",
    "https://about.gitlab.com/releases/2020/01/02/security-release-gitlab-12-6-2-released/"
  ],
  "aliases": [
    "CVE-2019-20142",
    "GSD-2019-20142"
  ],
  "assigner": "mitre",
  "epss": 0.11,
  "enisaIdProduct": [
    {
      "id": "4942d739-dfde-319a-9bae-24ba3733de93",
      "product": {
        "name": "n/a"
      },
      "product_version": "n/a"
    }
  ],
  "enisaIdVendor": [
    {
      "id": "f50922cb-4e8a-3421-b952-5ce57e4c1e44",
      "vendor": {
        "name": "n/a"
      }
    }
  ]
}